subprojects/build-option/src/test/groovy/org/gradle/internal/buildoption/BuildOptionTest.groovy

```diff
@@ -20,24 +20,21 @@ import org.gradle.cli.CommandLineArgumentException
 import spock.lang.Specification
 
 class BuildOptionTest extends Specification {
```

Comment:
:x: Needs to test the case we hint is null or blank. Also needs to test the use of the method that doesn't provide a hint parameter.

Reply:
Done.